dev/insights/docker-compose-ui.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
```

Comment:
Looks like insights and standalone docker-compose files are similar.
Can you extract parts that are common for both environments into common compose file?